engine/installation/linux/ubuntu.md

```diff
@@ -71,7 +71,7 @@ Docker from the repository.
 2.  Add Docker's official GPG key:
 
     ```bash
-    $ curl -s http://yum.dockerproject.org/gpg | sudo apt-key add
+    $ curl -s http://yum.dockerproject.org/gpg | sudo apt-key add -
```

Comment:
I notice it's using `http://` not `https://` as well, so at least that needs to be updated, although `apt-key` doesn't support that without `gnupg-curl` being installed, so if we go for that, we should add it to step 1.


If we _do_ stick with `curl`, we probably should change to;

```bash
$curl -fsSL https://yum.dockerproject.org/gpg | sudo apt-key add -
```